app/controllers/admin/tags/moderators_controller.rb

```diff
@@ -18,16 +18,14 @@ def create
           return redirect_to edit_admin_tag_path(params[:tag_id])
         end
 
-        notification_setting = user.notification_setting
-        if notification_setting.update(email_tag_mod_newsletter: true)
-          TagModerators::Add.call([user.id], [params[:tag_id]])
-          flash[:success] =
-            I18n.t("admin.tags.moderators_controller.added", username: user.username)
+        result = TagModerators::Add.call(user.id, params[:tag_id])
```

Comment:
Moved updating notifications settings to a service object to reuse the service object in `Admin::UsersController`.